README.md

```diff
@@ -29,23 +29,60 @@ download and install it yourself using the instructions found
 
 ## BUILD DEPENDENCIES & INSTRUCTIONS
 
-This project is build using [Apache Maven](https://maven.apache.org/) 3.6 
-(or later) and Java 11 OpenJDK (or later).  These prerequisites must be installed 
+### Maven
+
+This project can be built with [Apache Maven](https://maven.apache.org/) 3.6 
+(or later) and Java 11 OpenJDK (or later). These prerequisites must be installed 
 prior to building this project.  The following command can be used to download 
 all project dependencies and compile the Java module.  You can build this 
 project directly on a Raspberry Pi with Java 11+.  
 
-```text
+```
 mvn clean package
 ```
 
+### Gradle
+
+You can also use the [Gradle Build Tool](https://gradle.org/) from these same sources. Use version 6.6 (or later) 
+and Java 11 OpenJDK (or later).
+
+On Linux:
+
+```
+sdk install gradle
+gradle -v
+------------------
+Gradle 6.6
+------------------
+./gradlew build
```

Comment:
This mixes installing a system wide Gradle using SDKMAN and using `./gradlew` the Gradle Wrapper.
The instructions should stick to one single way of doing things. The preferred way of using Gradle is with the Gradle Wrapper, see https://docs.gradle.org/current/userguide/gradle_wrapper.html The wrapper files are already included in the sample build 👍 This will also ensure that everyone uses the same blessed version.

Reply:
sdk removed to avoid confusion + link to gradle userguide